src/js/components/helptips/helptooltips.js

```diff
@@ -145,3 +151,72 @@ const DeviceSupportTipComponent = ({ docsVersion }) => (
 );
 
 export const DeviceSupportTip = connect(mapStateToProps, actionCreators)(DeviceSupportTipComponent);
+
+const ConfigureTimezoneTipComponent = ({ anchor, device, toggleHelptips }) => {
+  if (!['qemux86-64', 'raspberry', 'rpi'].some(type => device.attributes?.device_type?.startsWith(type))) {
+    return null;
+  }
+  return (
+    <>
+      <div id="config-timezone-help" className="fadeIn tooltip help" data-tip data-for="config-timezone-tip" data-event="click focus" style={anchor}>
+        <HelpIcon />
+      </div>
+      <ReactTooltip id="config-timezone-tip" globalEventOff="click" place="bottom" type="light" effect="solid" className="react-tooltip">
+        <>
+          To see the effects of applying a configuration to your device you can set one of the below values to modify the timezone of your device. While all
+          values from <i>timedatectl list-timezones</i> will work, to easily see the impact of the changed value you can use one of the following values:
+          <ul>
+            <li>Europe/Oslo</li>
+            <li>America/Los_Angeles</li>
+            <li>Asia/Tokyo</li>
+          </ul>
+          Once the configuration has been applied you can see the effect by opening the Remote Terminal to the device and executing the <i>date</i> command.
+          <HideHelptipsButton toggleHelptips={toggleHelptips} />
+        </>
+      </ReactTooltip>
+    </>
+  );
+};
+
+export const ConfigureTimezoneTip = connect(mapStateToProps, actionCreators)(ConfigureTimezoneTipComponent);
+
+const ConfigureRaspberryLedComponent = ({ anchor, device, toggleHelptips }) => {
+  if (!['raspberry', 'rpi'].some(type => device.attributes?.device_type?.startsWith(type))) {
+    return null;
+  }
+  return (
+    <>
+      <div id="config-timezone-help" className="fadeIn tooltip help" data-tip data-for="config-timezone-tip" data-event="click focus" style={anchor}>
+        <HelpIcon />
+      </div>
+      <ReactTooltip id="config-timezone-tip" globalEventOff="click" place="bottom" type="light" effect="solid" className="react-tooltip">
+        <>
+          To see the effects of applying a configuration to your device you can set one of the below values to modify the behaviour of your Raspberry Pi status
+          LED
+          <ConfigurationObject
+            className="react-tooltip margin-top-small margin-bottom-small"
+            config={{
+              mmc0: 'The default, which blinks the led on storage activity',
+              on: 'Turn on the light permanently',
+              off: 'Turn off the light permanently',
+              heartbeat: 'Enable heartbeat blinking'
+            }}
+            compact
+          />
+          There are other possible values, but we won&apos;t advertise them here. See MEN-4461
+          <a href="http://www.d3noob.org/2020/07/controlling-activity-led-on-raspberry-pi.html" target="_blank" rel="noopener noreferrer">
+            in this blog post
+          </a>{' '}
```

Comment:
This sentence seems strange: "See MEN-4461 in this blog post or ..."?

What about: "See this blog post or ..." (the ticket reference is not important to get across here)

Reply:
thanks for catching that - it was not supposed to be there and is now gone 👌